### PR TITLE
nushell 0.9.0

### PR DESCRIPTION
**Pull request by waldyrious**
Created with `brew bump-formula-pr`.

Full commands used:

```sh
curl -Ls https://github.com/nushell/nushell/archive/0.9.0.tar.gz | sha256sum  | awk '{print $1}'
# f038e7c52fa1f43b372b1225e1da46569e0d61e02e1bd4d4dd6e018f00be025c

brew bump-formula-pr \
  --url=https://github.com/nushell/nushell/archive/0.9.0.tar.gz \
  --sha256=f038e7c52fa1f43b372b1225e1da46569e0d61e02e1bd4d4dd6e018f00be025c \
  nushell
```

### CODE REVIEW

**Comment by waldyrious** [comment]
I've changed the test to pass since [there's now a welcome message](https://www.nushell.sh/blog/2020/01/28/nushell-0_9_0.html#new-welcome-message-jonathandturner) shown when the shell is first started.

Is this an acceptable way to fix the test? I'm asking especially because the message includes the version, so the test will have to be updated on each formula bump. Could the test string skip the first line, or use a regex for the version part of it?

File: Formula/nushell.rb [hunk 1]
```diff
@@ -21,6 +21,7 @@ def install
   end
 
   test do
-    assert_equal "\n~ \n❯ 2\n\n~ \n❯ ", pipe_output("#{bin}/nu", 'echo \'{"foo":1, "bar":2}\' | from-json | get bar | echo $it')
+    assert_equal "Welcome to Nushell 0.9.0 (type 'help' for more info)\n~ \n❯ 2\n~ \n❯ ",
```
Comment (SMillerDev) [inline Formula/nushell.rb]: Please use the `#{version}` variable here

Comment (waldyrious) [reply Formula/nushell.rb]: Nice, thanks 👍 Will do in a bit.

Comment (waldyrious) [reply Formula/nushell.rb]: @SMillerDev done!

**Comment by chenrui333** [comment]
@BrewTestBot test this please

**Pull request closed without merging**